test/models/report_test.rb

```diff
@@ -3,7 +3,29 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  FIRST_REPORT = 0
+
+  def setup
+    @yocchan = create(:yocchan)
+    @takeru = create(:takeru)
```

Comment:
```suggestion
    @yocchan = create(:user)
    @takeru = create(:user)
```

別々のユーザーであることを保証できれば良いので、こんな感じで問題ないです